### DISCUSSION TITLE
Future of PySNMP

### DISCUSSION THREAD

**Discussion by daniel-fackrell-vasion** [pysnmp/pysnmp#41] [General]
I understand this fork wasn't originally intended to be the official successor to @etingof's work, so I'm wondering what the current status and plans might be.

Are the maintainers here willing to lobby to gain the ability to publish to PyPI? Would a rename make sense so this work can proceed in the absence of the original project's maintainer?

What related questions or concerns would need to be addressed?

**Comment by omrozowicz-splunk** [comment]
Hey,
You're right about this for wasn't intended to be official successor of the pysnmp repository. We've already had one discussion here: https://github.com/pysnmp/pysnmp/issues/34 - you can take a look. At the bottom @lextm pinned his issue from the official pysnmp repo where he listed the steps to do to bring pysnmp infrastructure back to life. He's the interested party to improve it more. We're open to share the ownership later on, include more maintainers etc. so the full project has a nice pysnmp organisation name.

**Comment by lextm** [comment]
You can refer to https://github.com/etingof/pysnmp/issues/429 for updates.

Most of the website setup (snmplabs.com -> pysnmp.com) has been done. The demo agent 161 is done. And I host the latest packages with `-lextudio` postfix temporarily.

To take over the original PyPI packages, I already sent a PEP 541 request by the end of November and now am waiting for its approval and execution.
